README.md

```diff
@@ -18,4 +18,16 @@ Then you need to run mkdocs from the root of that repository:
 
     $ mkdocs build
 
+Sometimes this might fail on Fedora 24+, so perform below steps
```

Comment:
Okay. I just realized that @ShyamsundarR had filed an [issue](https://github.com/gluster/glusterdocs/issues/213) about this.